traefik-compose.yaml

```diff
@@ -0,0 +1,56 @@
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: '2m'
+    max-file: '5'
+    mode: non-blocking
+
+services:
+  reverse-proxy:
+    image: traefik:${TRAEFIK_VERSION:-latest}
```

Comment:
I like all my services to have user-friendly container names:

```patch
    reverse-proxy:
      image: traefik:${TRAEFIK_VERSION:-latest}
+     container_name: traefik
      restart: ${RESTART_POLICY:-unless-stopped}
```

What do you think?